spec/openapi.yaml

```diff
@@ -2892,8 +2892,8 @@ paths:
             schema:
               $ref: '#/components/schemas/MatchParams'
         description: |-
-          To ensure context is mapped to the correct string, `stringHashcodes` or `contentFileUri` must be specified.
-          If both `stringHashcodes` and `contentFileUri` are not specified, **all strings from the project** will be included in the match.
+          To ensure context is mapped to the correct string, `stringHashcodes`, `contentFileUri` or `translationJobUids` must be specified.
+          If `stringHashcodes` or `contentFileUri` or `translationJobUids` are not specified, **all strings from the project** will be included in the match.
```

Comment:
>`stringHashcodes` or `contentFileUri` or `translationJobUids`
If none of `stringHashcodes`,`contentFileUri`, `translationJobUids` are specified, ....